libethereum/ChainParams.cpp

```diff
@@ -90,6 +90,41 @@ set<string> const c_knownParamNames = {c_minGasLimit, c_maxGasLimit, c_gasLimitB
     c_durationLimit, c_chainID, c_networkID, c_allowFutureBlocks};
 } // anonymous namespace
 
+void validateConfigJson(js::mObject const& _obj)
+{
+    requireJsonFields(_obj, "ChainParams::loadConfig",
+        {{"sealEngine", {json_spirit::str_type}}, {"params", {json_spirit::obj_type}},
```

Comment:
So why did you choose not to use the constants for the field names here?

Reply:
I guess this check is like a constant itself. if it does not match, then it throws an error. 
so if you have a typo, it will break the test run. 
although it is said that it is not good to use char in the code this way. maybe we need a dictionary class that would return a string by id for all text checks and messages in the client.

Reply:
We shouldn't have duplicate string literals in the code here, it's maintenance pain and might waste resources. So use here `c_sealEngine`, `c_params`  etc.